tests/yatai/local_yatai_service.py

```diff
@@ -47,7 +47,7 @@ def local_yatai_service_container(db_url=None, repo_base_url=None):
                 f"""\
 FROM bentoml/yatai-service:{LAST_PYPI_RELEASE_VERSION}
 ADD . /bentoml-local-repo
-RUN pip install /bentoml-local-repo
+RUN pip install -U /bentoml-local-repo
```

Comment:
@yubozhao FYI it looks like this was previously skipped because all packages are already installed in the base image - so this test might not be testing the latest code in yatai server